src/data/ACTIONS/AST.js

```diff
@@ -14,22 +14,366 @@ export default {
+		onGcd: true,
+		castTime: 2.5,
+	},
+
+	SYNASTRY: {
+		id: 3612,
+		name: 'Synastry',
+		icon: 'https://secure.xivdb.com/img/game/003000/003139.png',
+		cooldown: 90
+	},
+
+	COLLECTIVE_UNCON: {
+		id: 3613,
+		name: 'Collective Unconscious',
+		icon: 'https://secure.xivdb.com/img/game/003000/003140.png',
+		cooldown: 90
+	},
+	
+	MINOR_ARCANA: {
+		id: 7443,
+		name: 'Minor Arcana',
+		icon: 'https://secure.xivdb.com/img/game/003000/003106.png',
+		cooldown: 5
+	},
+
+	LADY_CROWNS: {
+		id: 7445,
+		name: 'Lady Of Crowns',
+		icon: 'https://secure.xivdb.com/img/game/003000/003146.png',
+		cooldown: 5
+	},
+
+	LORD_CROWNS: {
+		id: 7444,
+		name: 'Lord Of Crowns',
+		icon: 'https://secure.xivdb.com/img/game/003000/003147.png',
+		cooldown: 5
+	},
+
+	CELESTIAL_OPP: {
+		id: 3616,
+		name: 'Celestial Opposition',
+		icon: 'https://secure.xivdb.com/img/game/003000/003142.png',
+		cooldown: 120
+	},
+
+	TIME_DILATION: {
+		id: 3611,
+		name: 'Time Dilation',
+		icon: 'https://secure.xivdb.com/img/game/003000/003138.png',
+		cooldown: 90
 	},
 
 	EARTHLY_STAR: {
 		id: 7439,
 		name: 'Earthly Star',
 		icon: 'https://secure.xivdb.com/img/game/003000/003143.png',
+		cooldown: 60
 	},
-
+	
 	STELLAR_DETONATION: {
 		id: 8324,
 		name: 'Stellar Detonation',
 		icon: 'https://secure.xivdb.com/img/game/003000/003144.png',
+	}, 
+
+	DRAW: {
+		id: 3590,
+		name: 'Draw',
+		icon: 'https://secure.xivdb.com/img/game/003000/003101.png',
+		cooldown: 30
+	},
+
+	ROYAL_ROAD: {
+		id: 3591,
+		name: 'Royal Road',
+		icon: 'https://secure.xivdb.com/img/game/003000/003103.png',
+		cooldown: 15
+	},
+
+	SPREAD: {
+		id: 3592,
+		name: 'Spread',
+		icon: 'https://secure.xivdb.com/img/game/003000/003104.png',
+		cooldown: 30
+	},
+
+	REDRAW: {
+		id: 3593,
+		name: 'Redraw',
+		icon: 'https://secure.xivdb.com/img/game/003000/003105.png',
+		cooldown: 30
+	},
+
+	SLEEVE_DRAW: {
+		id: 7448,
+		name: 'Sleeve Draw',
+		icon: 'https://secure.xivdb.com/img/game/003000/003107.png',
+		cooldown: 120
+	},
+
+	// -----
+	// Arcana cast from draw slot
+	// -----
+
+	THE_BALANCE: {
+		id: 4401,
+		name: 'The Balance',
+		icon: 'https://secure.xivdb.com/img/game/003000/003110.png',
+		cooldown: 30
+	},
+
+	THE_BOLE: {
+		id: 4402,
+		name: 'The Bole',
+		icon: 'https://secure.xivdb.com/img/game/003000/003111.png',
+		cooldown: 30
+	},
+
+	THE_ARROW: {
+		id: 4403,
+		name: 'The Arrow',
+		icon: 'https://secure.xivdb.com/img/game/003000/003112.png',
+		cooldown: 30
+	},
+
+	THE_SPEAR: {
+		id: 4404,
+		name: 'The Spear',
+		icon: 'https://secure.xivdb.com/img/game/003000/003113.png',
+		cooldown: 30
+	},
+
+	THE_EWER: {
+		id: 4405,
+		name: 'The Ewer',
+		icon: 'https://secure.xivdb.com/img/game/003000/003114.png',
+		cooldown: 30
+	},
+
+	THE_SPIRE: {
+		id: 4406,
+		name: 'The Spire',
+		icon: 'https://secure.xivdb.com/img/game/003000/003115.png',
+		cooldown: 30
+	},
+
+	// -----
+	// Expanded arcana cast from draw slot
+	// -----
+
+	THE_BALANCE_EXPANDED: {
```

Comment:
Today I learned that square has different action IDs for all the royals. What the flying fuck.

Reply:
That's what I thought at first, but it's not as bad! They actually they only differentiate between Expanded/Single-target, and then another set for Cast from draw or Cast from spread. I wonder if this will affect your Arcanum.js in core